scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1730,15 +1730,12 @@ class Router(formatOps: FormatOps) {
           ) NoPolicy
           else defaultPolicy
 
+        val foo = if (bodyBlock) 0 else 2
```

Comment:
my apologies, i didn't mean that the variable should be named `foo`, i just left it to you to provide a meaningful name.

Reply:
I was unable to come up with one 😀

Could be also simple `a` or `i`, what do you think?

Reply:
variables may not have cryptic, meaningless names. how about `entireClauseIndent`?